js/main.js

```diff
@@ -189,21 +207,11 @@ $('body').on('keypress', 'input', function(event) {
 
 //keyboard shortucts
 $(window).bind('keydown', function(e) {
-  var char = String.fromCharCode(e.which).toLowerCase(),
-      ctrl = (e.ctrlKey || e.metaKey) && !e.altKey, //test for alt key to prevent international keyboard issues
-      route = null;
-    
-  if (event.keyCode == 116) { //on F5 press
-    Backbone.history.loadUrl();
-  }
+  if ((e.ctrlKey || e.metaKey) && !e.altKey) { //test for alt key to prevent international keyboard issues
+    var route = null,
+        char = String.fromCharCode(e.which).toLowerCase();
 
-  if (ctrl) {
     switch (char) {
-      case config.keyShortcuts.undo:
-        //run undo programmatically to avoid crash
-        e.preventDefault();
-        document.execCommand('undo');
```

Comment:
Did the control-z problem get fixed somewhere else, or is it no longer a bug in electron?